drivers/hdc1000/hdc1000.c

```diff
@@ -86,36 +87,48 @@ void hdc1000_trigger_conversion(hdc1000_t *dev)
      * to the address 0x00 (HDC1000_TEMPERATURE).
      * Conversion Time is 6.50ms for each value for 14 bit resolution.
      */
-    i2c_write_byte(dev->p.i2c, dev->p.addr, HDC1000_TEMPERATURE);
+    if (i2c_write_byte(dev->p.i2c, dev->p.addr, HDC1000_TEMPERATURE) != 1) {
+        status = HDC1000_BUSERR;
```

Comment:
why not simply `return HDC100_BUSERR`? Omitting `int status`, though I think compilers will optimize that away anyway.